en/docs/how-tos/passing-enduser-attributes-to-the-backend-using-jwt.md

```diff
@@ -0,0 +1,186 @@
+# Passing Enduser Attributes to the Backend Using JWT 
+
+You can use JWT Generation in API Microgateway to send a customized JWT to the backend with user preferred claims. You have 2 possible options when selecting a method to send a JWT to the backend.
```

Comment:
Better if we can add more descriptive description of what backend jwt. That should go in to concepts section. Thats something we can do later, but keep a note of it or create an issue.

Reply:
We can get some content for this from here https://apim.docs.wso2.com/en/3.0.0/learn/api-gateway/passing-end-user-attributes-to-the-backend/passing-enduser-attributes-to-the-backend-using-jwt/#passing-enduser-attributes-to-the-backend-using-jwt

Reply:
Issue created for the requirement https://github.com/wso2/docs-mg/issues/33